infrastructure/src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -102,11 +101,19 @@
 
     private static final String metricsPrefix = "corfu.server.logunit.";
 
+    private final ServerContext serverContext;
+
```

Comment:
The base server is not shared among all component servers.
Using the serverContext management server can signal all servers to start accepting requests.